File: blog-sever/server-java/src/main/java/com/site/blog/model/vo/BlogEditVO.java
package com.site.blog.model.vo;


import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.Data;

import java.time.LocalDateTime;

import java.util.List;

@Data
public class BlogEditVO {
    private String blogId;

    private String blogTitle;

    private String blogCategoryId;

    private Long commentCount;

    private String blogCategoryIcon;

    private Boolean show;
    private String subUrl;
    private String preface;
    private String blogCoverImage;

    private Long blogViews;

    private List<String> blogTagIds;

    private String blogContent;

    private Boolean enableComment;
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", timezone = "GMT+8")
    private LocalDateTime createTime;

}

File: blog-sever/server-kt/src/main/kotlin/com/site/blog/model/vo/BlogEditVO.kt
package com.site.blog.model.vo

import com.fasterxml.jackson.annotation.JsonFormat
import java.time.LocalDateTime

data class BlogEditVO  (
    var  blogId: String? = null,
    var  blogTitle: String? = null,
    var  blogCategoryId: String? = null,
    var  commentCount: Long? = null,
    var  blogCategoryIcon: String? = null,
    var  show: Boolean? = null,
    var  subUrl: String? = null,
    var  preface: String? = null,
    var  blogCoverImage: String? = null,
    var  blogViews: Long? = null,
    var  blogTagIds: List<String?> = emptyList(),
    var  blogContent: String? = null,
    var  enableComment: Boolean? = null,

    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", timezone = "GMT+8")
    var  createTime: LocalDateTime? = null,
)